src/components/issues/ActionForm.tsx

```diff
@@ -43,12 +44,26 @@ const reducer = (state: ActionFormReducer, action: reducerAction) => {
 
 const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
     const [state, dispatch] = useReducer(reducer, initialState);
+    const [isAssigned, setIsAssigned] = useState(false);
     const [updateTask] = useUpdateTaskMutation();
+    const { data } = useGetTaskDetailsQuery(taskId);
+
+    useEffect(() => {
```

Comment:
why do you need an useEffect for this ?

Reply:
it was giving the multiple re rendering error